README.md

```diff
@@ -195,6 +195,14 @@ options:
   # commonly created by using the "Update branch" button in the UI.
   ignore_update_merges: false
 
+  # Automatically request reviewers when a Pull Request is opened
+  # if this rule is pending, there are no assigned reviewers, and if the
+  # Pull Request is not in Draft. Reviewers are selected randomly
+  # based on the set of requirements for this rule.
```

Comment:
This is good for a config description, but this feel like a feature that would also benefit from a larger section in the README, particularly since the behavior is kind of subtle once you consider when it applies, the reviewers have to be contributors, and any future extensions around and/or.